tests/scripts/test_scripts.py

```diff
@@ -95,9 +93,6 @@ def test_main_console(script_mod):
 
 PREFERENCE_COMPARISON_CONFIGS = [
     {},
-    {
-        "trajectory_path": CARTPOLE_TEST_ROLLOUT_PATH,
```

Comment:
Are we no longer testing ability to load rollouts from a file in script? Might be good to retain some coverage of that (new key `demonstrations.rollout_path` I believe)

Reply:
I added this case back.